cpp/arcticdb/util/trace.cpp

```diff
@@ -9,46 +9,12 @@
 
 #ifndef  _WIN32
 #include <cxxabi.h>
-#include <execinfo.h>
 #endif
 
 #include <iostream>
-#include <sstream>
 
 namespace arcticdb {
 
-    std::string get_stack() {
-#ifndef _WIN32
-        char sep = '\n';
-        void *trace[256];
-        int trace_size = ::backtrace(trace, 256);
-        char **buffer = ::backtrace_symbols(trace, trace_size);
-
-        std::string stack;
-        for (int i = 1; buffer && i < trace_size - 1; i++) {
-            stack += "[#" + std::to_string((trace_size - 1) - i) + "] ";
-            std::string s = buffer[i];
-            std::string mangle = s.substr(s.find('(') + 1, s.find('+') - s.find('(') - 1);
-
-            if (char *demangled = abi::__cxa_demangle(mangle.c_str(), nullptr, nullptr, nullptr)) {
-                std::string dm = demangled;
-                free(demangled);
-                stack += dm;
-            } else stack += mangle;
-
-            stack += sep;
-        }
-
-        if (buffer)
-            free(buffer);
-
-        return stack;
-#else
-        //TODO: Implement get_stack for windows
-        return std::string("");
-#endif
-    }
-
     std::string get_type_name(const std::type_info & ti){
```

Comment:
Given this lacks implementations for most platforms we (are going to) support, we should switch to an [implementation ](https://github.com/facebook/folly/blob/main/folly/Demangle.cpp) from a library and rename the file?

Reply:
Not in scope